code/style.css

```diff
@@ -1,4 +1,314 @@
-html {
-  background: #0025ff;
+* {
+ padding: 0px; 
+ margin: 0px;
+
+}
+
+
+/* Header with logo and nav */
+header {
+  display:flex;
+  width: 100%;
+  justify-content: space-between;
+  z-index: 1;
+
+  background-color: #fff;
+}
+
+.logoIcon  {
+  width: 140px;
+  margin-left: 30px;
+  margin-right: 50px;
+  margin-top: 20px;
+  margin-bottom: 30px;
```

Comment:
Since you're using every part of the margin property here you can style it with just margin. Is has four values that goes clockwise, from the top to the right to the bottom to the left.
In this case it could look like this instead:
```
margin: 20px 50px 30px 30px;
```
Saves you a bit of code and is nice to know for future use.